### PR TITLE
create report template w/o name

### PR DESCRIPTION
**Pull request by amolpati30**
### Problem Statement
Create template report with blank name

### Solution
Given solution will not create template report without name, it should give error.

### Related Issues
No

### CODE REVIEW

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_reporttemplates.py::test_negative_create_report_without_name

File: tests/foreman/api/test_reporttemplates.py [hunk 1]
```diff
@@ -378,6 +377,10 @@ def test_negative_create_report_without_name():
 
     :CaseImportance: Medium
     """
+    template1 = gen_string('alpha')
+    with pytest.raises(HTTPError) as report_response:
+        module_target_sat.api.ReportTemplate(name=' ', template=template1).create()
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_reporttemplates.py:382]:
```suggestion
    with pytest.raises(HTTPError) as report_response:
        module_target_sat.api.ReportTemplate(name=' ', template=gen_string('alpha')).create()
```

Comment (amolpati30) [reply tests/foreman/api/test_reporttemplates.py]: Thank you for your suggestion. changes accepted.

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5690
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/api/test_reporttemplates.py::test_negative_create_report_without_name
Test Result : ============ 2 passed, 404 warnings in 3767.31s (1:02:47) ============
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5713
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/api/test_reporttemplates.py::test_negative_create_report_without_name
Test Result : ================== 1 passed, 11 warnings in 731.17s (0:12:11) ==================
```

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**